phing/build.yml

```diff
@@ -58,9 +60,9 @@ drupal:
 
 drush:
   bin: ${composer.bin}/drush
-  cmd: ${drush.bin} @${drush.alias} -r ${docroot} -l ${multisite.name}
+  cmd: ${drush.bin} @${drush.alias} -r ${docroot} -l ${site.uri.${environment}}
```

Comment:
@danepowell - yeah, perhaps the `uri` argument should go away entirely and the alias argument should be multisite aware within BLT (something along the lines of `@site.drush.aliases.${environment}`). This is the approach that I've tried to take in managing previous multisite projects (i.e. an alias for each site environment) but currently BLT seems to favor using the `uri` argument. This works fine except when trying to do a `sql-sync ` which requires two aliases if you want to sync remote to local. 

This being the case, the PR tries to keep both intact, but making alias configurations responsible for this may be the cleaner approach. 

What do you think?